src/app/register/register.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="container">
+	<div class="col-xs-2">
+	</div>
```

Comment:
That is to prevent the form from taking up whole screen.  Will use offset instead.